src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,28 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info): If you create a formatter, make it a constant field.